src/fsharp/vs/Symbols.fs

```diff
@@ -1688,6 +1688,32 @@ and FSharpType(cenv, typ:TType) =
        protect <| fun () -> 
         "type " + NicePrint.stringOfTy (DisplayEnv.Empty(cenv.g)) typ 
 
+    member x.Symbol =
```

Comment:
Well, yes, now that you mention it! So I don't think you need this PR, you just use fSharpType.TypeDefinition guarded by HasTypeDefinition